README.md

```diff
@@ -21,13 +21,13 @@ Stage 2 indicates that the committee expects these features to be developed and
 | [Hashbang Grammar][hashbang-grammar]                                           | Bradley Farias                                                          | Bradley Farias                                                          | [:white_check_mark:][tests-hashbang-grammar]     | <sub>[November&nbsp;2018][hashbang-notes]</sub>            |
 | [`Atomics.waitAsync`][nonblocking]                                             | Lars Hansen                                                             | Shu-yu Guo<br />Lars Hansen                                             | [:white_check_mark:][tests-nonblocking]          | <sub>[December&nbsp;2019][nonblocking-notes]</sub>         |
 | [`.at()`][at]                                                                  | Shu-yu Guo<br />Tab Atkins                                              | Shu-yu Guo<br />Tab Atkins                                              | [:white_check_mark:][tests-at]                   | <sub>[November&nbsp;2020][at-notes]</sub>                  |
+| [Accessible `Object.prototype.hasOwnProperty`][object-has]                     | Jamie Kyle                                                              | Tierney Cyren<br />Jamie Kyle                                           | [:white_check_mark:][tests-object-has]           | <sub>[July&nbsp;2021][object-has-notes]</sub>              |
```

Comment:
these are generally sorted by "stage insertion order"; i can move it back for you.

Reply:
Oh, thought all the ones with tests were supposed to be at the top

Reply:
nah that's just because the older ones tend to have tests :-)